src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -2,56 +2,68 @@
+| ----------------------------- | --------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Manufacturer                  | RECOMMENDED           | [string][]    | Manufacturer of the equipment that produced the composite instances. Corresponds to DICOM Tag 0008, 0070 `Manufacturer`                                                                                                                                                   |
+| ManufacturersModelName        | RECOMMENDED           | [string][]    | Manufacturer's model name of the equipment that produced the composite instances. Corresponds to DICOM Tag 0008, 1090 `Manufacturers Model Name`                                                                                                                          |
+| DeviceSerialNumber            | RECOMMENDED           | [string][]    | The serial number of the equipment that produced the composite instances. Corresponds to DICOM Tag 0018, 1000 `DeviceSerialNumber`. A pseudonym can also be used to prevent the equipment from being identifiable, so long as each pseudonym is unique within the dataset |
+| StationName                   | RECOMMENDED           |               | Institution defined name of the machine that produced the composite instances. Corresponds to DICOM Tag 0008, 1010 `Station Name`                                                                                                                                         |
+| SoftwareVersions              | RECOMMENDED           | [string][]    | Manufacturer's designation of software version of the equipment that produced the composite instances. Corresponds to DICOM Tag 0018, 1020 `Software Versions`                                                                                                            |
+| HardcopyDeviceSoftwareVersion | [DEPRECATED][]        | [string][]    | Manufacturer's designation of the software of the device that created this Hardcopy Image (the printer). Corresponds to DICOM Tag 0018, 101A `Hardcopy Device Software Version`                                                                                           |
+| MagneticFieldStrength         | RECOMMENDED           |               | Nominal field strength of MR magnet in Tesla. Corresponds to DICOM Tag 0018,0087 `Magnetic Field Strength`                                                                                                                                                                |
```

Comment:
what datatype is this?

Reply:
Number. In OpenNeuro, it's mostly numbers, though there are a couple strings like `"3T"` and `" "`. I would call those invalid, though.

Reply:
yarik seems to agree: https://github.com/bids-standard/bids-validator/issues/851